MockKit/Extensions/Collection.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Dispatch
+import LoopKit
```

Comment:
This import addition shouldn't be necessary; the extension method in this file is agnostic to LoopKit data types.

Reply:
I made Locked public in LoopKit, since we're using it it a lot of places that use LoopKit.